src/bandersnatch/tests/plugins/test_storage_plugins.py

```diff
@@ -586,6 +586,8 @@ class BaseStoragePluginTestCase(BasePluginTestCase):
 web{0}simple{0}index.html""".format(
         os.sep
     ).strip()
+    if os.name == "nt":
+        base_find_contents = base_find_contents.replace(".lock\n", "")
```

Comment:
FWIW mypy prefers `sys.platform == "win32"`

Reply:
sigh. Is there any difference